v23.1/cdc-queries.md

```diff
@@ -112,22 +128,40 @@ CREATE CHANGEFEED INTO "scheme://sink-URI" WITH updated, schema_change_policy =
 
 ### Filter delete messages
 
-To remove the [delete messages](changefeed-messages.html#delete-messages) from a changefeed stream, use the [`cdc_is_delete()`](#cdc-transformation-function-support) function:
+To remove the [delete messages](changefeed-messages.html#delete-messages) from a changefeed stream, use the [`event_op()`](#cdc-query-function-support) function:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE CHANGEFEED INTO sink AS SELECT * FROM table WHERE NOT event_op() = 'delete';
+~~~
+
+Filtering delete messages from your changefeed is helpful for certain outbox table use cases. See [Queries and the outbox pattern](#queries-and-the-outbox-pattern) for further detail.
+
+### Emit the previous state of a row
+
+{% include_cached new-in.html version="v23.1" %} Changefeeds can access the `cdc_prev` hidden column on a table to emit the previous state of a row or column. `cdc_prev` is a tuple-typed column that contains the table's columns. 
+
+To emit the previous state of a row, it is necessary to explicitly call `cdc_prev`:
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-CREATE CHANGEFEED INTO sink WITH schema_change_policy = 'stop' AS SELECT * FROM table WHERE NOT cdc_is_delete();
+CREATE CHANGEFEED INTO 'external://sink' AS SELECT *, cdc_prev FROM table;
 ~~~
 
-Filtering delete messages from your changefeed is helpful for certain outbox table use cases. See [Transformations and the outbox pattern](#transformations-and-the-outbox-pattern) for further detail.
+To emit the previous state of a column, you can specify this as a named field from the `cdc_prev` tuple with the following syntax:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE CHANGEFEED INTO 'external://sink' AS SELECT *, cdc_prev FROM table WHERE (cdc_prev).column = 'value';
```

Comment:
Question: is `value` correct here? The remainder of examples use actual values, so a seeming-placeholder seems a little out of place. Alternatively, I could absolutely be deeply mistaken.

Reply:
It is correct, but I do see your point that it doesn't follow the more pointed, named examples from the other examples. I have updated this with the movr example I tested!

Reply:
Thanks!